algorithms/gr-dij/csharp/DijkstraAlgorithm.cs

```diff
@@ -0,0 +1,92 @@
+// A C# program for Dijkstra's  shortest path algorithm. 
+using System; 
+
+class Dijkstra { 
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6349348)